game/tests.py

```diff
@@ -229,51 +230,43 @@ def setUp(self):
         self.ct_item = ContentType.objects.get(model='item').pk
         self.ct_knowledge = ContentType.objects.get(model='knowledge').pk
         self.ct_rol = ContentType.objects.get(model='rol').pk
-
-    def test_player_join_to_game_without_autentication(self):
-        game_id = Game.objects.first().pk
-        data = {
+        self.data = {
             'position': {
                 'longitude': 37.201421,
                 'latitude': -6.9447224
             }
         }
-        response = self.client.post('/game/{}/join/'.format(game_id), data)
+
+    def test_player_join_to_game_without_autentication(self):
+        response = self.client.post('/game/{}/join/'.format(self.game.pk),
+                                    self.data)
         self.assertEqual(response.status_code, 401)
 
     def test_player_join_to_game_content_not_exist(self):
         self.authenticate()
-        game_id = Game.objects.first().pk
-        data = {
-            'position': {
-                'longitude': 37.201421,
-                'latitude': -6.9447224
-            }
-        }
-        response = self.client.post('/game/{}/join/'.format(game_id), data)
+        response = self.client.post('/game/{}/join/'.format(self.game.pk),
+                                    self.data)
         self.assertEqual(response.status_code, 201)
 
     def test_player_join_to_game_content_exist(self):
         self.authenticate()
-        game_id = Game.objects.first().pk
-        data = {
-            'position': {
-                'longitude': 37.201421,
-                'latitude': -6.9447224
-            }
-        }
-        self.client.post('/game/{}/join/'.format(game_id), data)
```

Comment:
avoid create content with api, if create content by api fail, this test fail too